client/src/components/StepsQuestions/index.css

```diff
@@ -0,0 +1,74 @@
+/* ================================= */
+/* Basic style */
+/* ================================= */
+
+.steps {
+  margin-top: 4rem;
+  padding: 3rem 0;
+}
+
+.steps-content {
+  min-height: 40rem;
+  margin-top: 16px;
+  border: 1px dashed #e9e9e9;
+  border-radius: 6px;
+  background-color: #fafafa;
+  padding: 1.6rem 0.4rem;
+}
+
+.title {
```

Comment:
`.title` is a pretty generic className so it would be safer to add the .steps class in front of this selector, to guarantee that these styles only apply to the step component's title (and not every other .title that might appear in the app).  So like this: `.steps .title {`